<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudjetUpdateDlg</class>
 <widget class="QDialog" name="BudjetUpdateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Бюджет</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="toolTip">
      <string/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Опции учёта</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkChildren">
           <property name="toolTip">
            <string>Учитывать всю ветку категорий</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Потомки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkWorkDays">
           <property name="toolTip">
            <string>Учитывать только рабочие дни</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Рабочие дни</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkCorrection">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;Корректировать отклонения &lt;br/&gt;на следующий период&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Корректировка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkIllness">
           <property name="toolTip">
            <string>Учитывать дни болезней</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Болезни</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Ограничения</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Действует по:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dtEnd"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Период:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboPeriod"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>203</height>
           </size>
          </property>
          <property name="title">
           <string>Бюджет</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>менее</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtLow"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>по</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtHigh"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>более</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Тип</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="rightMargin">
            <number>35</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Тип:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboType">
             <property name="minimumSize">
              <size>
               <width>198</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>-1</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Выбор:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboChoice">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>202</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnLookup">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../TM3res.qrc">
               <normaloff>:/images/Basic_set_Png/search_32.png</normaloff>:/images/Basic_set_Png/search_32.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lblCat">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>Наименование категории</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="txtName"/>
      </item>
     </layout>
     <zorder>txtName</zorder>
     <zorder>label</zorder>
     <zorder>groupBox</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../TM3res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BudjetUpdateDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BudjetUpdateDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BudjetUpdateDlg</receiver>
   <slot>typeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>-57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboChoice</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BudjetUpdateDlg</receiver>
   <slot>choiceChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLookup</sender>
   <signal>clicked()</signal>
   <receiver>BudjetUpdateDlg</receiver>
   <slot>catLookup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>848</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>periodChanged(int)</slot>
  <slot>typeChanged(int)</slot>
  <slot>choiceChanged(int)</slot>
  <slot>catLookup()</slot>
 </slots>
</ui>
